kubernetes/service-monitor-for-prometheus.yaml

```diff
@@ -36,4 +36,3 @@ spec:
       app: apisix-gw
   namespaceSelector:
     any: true
-
```

Comment:
🤔 May need to keep the last line?

Reply:
Last line is there seen here:

view-source:https://raw.githubusercontent.com/apache/apisix/9f771b29232929dc5e85b9c97c4d43ff86924ee5/kubernetes/service-monitor-for-prometheus.yaml

That file has 2 lines at the end